backend/TreeOfAKind.Infrastructure/Database/TreesContext.cs

```diff
@@ -8,18 +9,18 @@ namespace TreeOfAKind.Infrastructure.Database
     public class TreesContext : DbContext
     {
         public DbSet<Tree> Trees { get; set; }
```

Comment:
That doesn't seem like something that should have `set`, probably `public DbSet<Tree> Trees => Set<Tree>()` would be better.